src/main/java/uk/ac/ebi/eva/pipeline/io/writers/AnnotationMongoWriter.java

```diff
@@ -86,10 +86,12 @@ public AnnotationMongoWriter(MongoOperations mongoOperations, String collection)
     }
 
     @Override
-    public void write(List<? extends Annotation> annotations) throws Exception {
+    public void write(List<? extends List<Annotation>> annotations) throws Exception {
         BulkOperations bulk = mongoOperations.bulkOps(BulkOperations.BulkMode.UNORDERED, collection);
```

Comment:
Not sure of this, originally this was sound but if you only initialize once, I don't know if we can expect the underlying implementation to clean the elements in the bulk after each execute operation.